web/react/components/mention_list.jsx

```diff
@@ -20,6 +20,10 @@ module.exports = React.createClass({
         PostStore.addMentionDataChangeListener(this._onChange);
         var self = this;
 
+        $('.post-right__scroll').scroll(function(){
+            self.forceUpdate();
```

Comment:
What's the reason for the forceUpdate? Doing a forceUpdate every time someone scrolls could be very expensive.